tests/providers/amazon/aws/hooks/test_glue.py

```diff
@@ -30,6 +30,9 @@
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 from airflow.providers.amazon.aws.hooks.glue import GlueJobHook
 
+if TYPE_CHECKING:
+    from unittest.mock import MagicMock
```

Comment:
This is honestly somewhat awkward, we could just write `mock.MagicMock` without any additional imports…

Reply:
I guess the root issue why this import even exists that PyCharm (and maybe others IDE) can't correctly use annotation  from `@mock.patch` and `@mock.patch.object`, so you need to set in explicitly if you want to use autosuggestion/autocompletion from the IDE.

I found for myself that use fixtures do not required to manually annotate `mock.patch` for IDE
```python
@pytest.fixture
def awesome_fixture():
    with mock.patch("foo.bar.spam.Egg") as m:
        yield m
```